daemon/logger/loggerutils/rotatefilewriter.go

```diff
@@ -120,5 +120,10 @@ func (w *RotateFileWriter) NotifyRotateEvict(sub chan interface{}) {
 
 // Close closes underlying file and signals all readers to stop.
 func (w *RotateFileWriter) Close() error {
+	w.mu.Lock()
```

Comment:
It seems like a good idea to take the lock here. It's not clear to me if `Close` is called concurrently with `Write`, but why take the chance?

However, shouldn't `LogPath` also lock the mutex?